opentelemetry-intro-course/src/otelcol/metrics.md

```diff
@@ -1 +1,60 @@
 # Metrics
+
+## Manual Instrumentation 
+The python apps part of the framework are instrumented with ManualTelemetry-Python. These metrics are configured using statsd . 
+To receive the Statsd metrics you can configure it in otelcol.yaml receivers. The otelcol.yaml can be found at deployments/docker-compose.
+
+Sample Configuration for Python App:
+
+  statsd:
+    endpoint: "0.0.0.0:8125" # default
+    aggregation_interval: 60s  # default
+    enable_metric_type: true   # default
+    is_monotonic_counter: true # default
+    timer_histogram_mapping:
+      - statsd_type: "histogram"
+        observer_type: "gauge"
+      - statsd_type: "timing"
+        observer_type: "gauge"
+
+Samples of metrics generated in Sumo portal : 
+
+    {
+    _collector="tcb-otc-distro",
+    _collectorId="000000000*****",
+    _source="tcb-otc-distro",
+    _sourceCategory="tcb-otc-distro",
+    _sourceHost="otelcol",
+    _sourceId="0000000000000000",
+    _sourceName="OTC Metric Input"
+    metric_type="counter"
+    }
+
+## Auto Instrumentation for Python App:
```

Comment:
OpenTelemetry Metrics are not "auto instrumentation metrics". They have to be defined as you presented in the code below. So metric has to be defined and then it has to be updated somewhere in the code.